cumulusci/cli/cci.py

```diff
@@ -563,6 +563,15 @@ def project_init(runtime):
         test_name_match = None
     context["test_name_match"] = test_name_match
 
+    context["code_coverage"] = None
+    if click.confirm(
+        click.style("Do you want to enforce Apex code coverage?", bold=True),
+        default=True,
+    ):
+        context["code_coverage"] = click.prompt(
+            click.style("Minimum code coverage percentage", bold=True), default=75
+        )
```

Comment:
@davidmreed @jstvz Is this just going to mislead people into thinking they can change what coverage the _platform_ requires for their managed package?

How about: Remove the prompt. Set this to 75 if the user answered yes to the "managed package" question, otherwise don't.

Reply:
I’m not sure I entirely agree. The platform enforces 75% code coverage for any production deployment. I suggest warning the user when coverage <75% instead.

Reply:
Isn't the point to break feature builds that have insufficient coverage though? I think a warning would get lost.

I guess I'm fine with including this prompt here, but I think it would also be okay to just omit it, let the option default to 75, and users can override it if they aren't developing for a production environment.

Reply:
The warning would be here, in the context of the wizard, not builds. 

Omitting it works too.

Reply:
I changed the initial prompt to try to clarify what we're actually doing here. 

> Do you want to check Apex code coverage when tests are run?

Does that help cover your concerns, @davisagli? I do want to be clear to users that what they're opting in to is a CCI behavior (a build failure), not anything at the platform level.